packages/core/admin/shared/contracts/admin.ts

```diff
@@ -0,0 +1,85 @@
+// TODO replace SettingsFile with type from the upload plugin ?
+interface Logo {
+  name: string;
+  url: string;
+  width: number;
+  height: number;
+  ext: string;
+  size: number;
+}
+
+export type SettingsFile = any;
+
+/**
+ * /init - Initialize the admin panel
+ */
+export declare namespace Init {
+  export interface Response {
+    data: {
+      uuid: string | false;
+      hasAdmin: boolean;
+      menuLogo?: SettingsFile;
+      authLogo?: SettingsFile;
+    };
+  }
+}
+
+/**
+ * /project-settings - Get and modify the project settings
+ */
+export declare namespace ProjectSettings {
+  export interface Response {
+    menuLogo: SettingsFile;
+    authLogo: SettingsFile;
+  }
+}
+
+/**
+ * /information - get project information
+ */
+export declare namespace Information {
+  export interface Response {
+    data: {
+      currentEnvironment: string;
+      autoReload: boolean;
+      strapiVersion: string | null;
+      dependencies: Record<string, string>;
+      projectId: string | null;
+      nodeVersion: string;
+      communityEdition: boolean;
+      useYarn: boolean;
+    };
+  }
+}
+
+/**
+ * /telemetry-properties - get telemetry properties
+ */
+export declare namespace TelemetryProperties {
+  export interface Response {
+    data: {
+      useTypescriptOnServer: boolean;
+      useTypescriptOnAdmin: boolean;
+      isHostedOnStrapiCloud: boolean;
+      numberOfAllContentTypes: number;
+      numberOfComponents: number;
+      numberOfDynamicZones: number;
+    };
+  }
+}
+
+/**
+ * /plugins - get plugin information
+ */
+export declare namespace Plugins {
+  // TODO should this type come from elsewhere ?
+  interface Plugin {
```

Comment:
Would it not make more sense to make it come from https://github.com/strapi/strapi/blob/0cd16805723f7aaa37cc77ce2177c7fa84b46753/packages/core/strapi/src/core/loaders/plugins/index.ts#L11?

Reply:
doesn't seem we have such definition elsewhere, as its a simple type lets keep it here

Reply:
We should import from  strapi types as much as possiblel instead of other packages. So if we want to define this elsewhere I would add it there.

Having said this, this is a very simple contract and I woulnd't mind keeping it like this.

Reply:
As this seems to be the only plugin related controller I think this type will only be needed here so I've kept it defined in line